### ISSUE TITLE
Its broken

### ISSUE THREAD

**Issue by Listlessness** [b31ngd3v/aniflix#6]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by b31ngd3v** [comment]
yes, ik. but I can do nothing. gogoanime adds some breaking changes every week or so. and it'll be very hard for me to maintain the project.

**Comment by Listlessness** [comment]
Use a diffrent service. That's not go anime.
